onnxruntime/python/tools/transformers/models/llama/benchmark_e2e.py

```diff
@@ -103,6 +117,18 @@ def get_model(args: argparse.Namespace):
     return model
 
 
+# When it runs a model without position_ids input, ORT keep printint out a complaint.
+# Check if a model has a position_ids input and suppress if not.
+def has_position_ids(args):
```

Comment:
Since the warning message is printed from [here](https://github.com/microsoft/onnxruntime/blob/dc545d366dc2d139b8585c2acb2c8dedf519b0b6/onnxruntime/python/tools/transformers/models/llama/llama_inputs.py#L303), should we just remove the message instead of adding the conditional logic for position ids? If not, can we move this method to `llama_inputs.py` so it can be used by other scripts in this folder?

Reply:
I thought the warning message comes from ORT. Definitely it's better to remove the message from the script.